apps/website/src/components/OpenInCodePen.astro

```diff
@@ -0,0 +1,120 @@
+---
+import outdent from 'outdent';
+import VisuallyHidden from './utils/VisuallyHidden.astro';
+
+type Props = { code: string } & astroHTML.JSX.HTMLAttributes;
+const { code = 'export default () => "hello"', ...rest } = Astro.props;
+
+const css = outdent`
+  @layer reset, itwinui;
+
+  .app-wrapper {
+    height: 100dvh;
+    padding: 1rem;
+  }
+
+  @layer reset {
+    * {
+      box-sizing: border-box;
+    }
+    body {
+      margin: 0;
+    }
+  }
+`;
+const js = outdent`
+  ${code.replace('export default () =>', 'const App = () =>')}
+
+  // -------------------------------------------------------------
+
+  import { createRoot } from 'react-dom/client';
+  import { ThemeProvider } from '@itwin/itwinui-react';
+
+  createRoot(document.body).render(
+    <ThemeProvider className='app-wrapper'>
+      <App />
+    </ThemeProvider>
+  );
+`
+  .replaceAll(`@itwin/itwinui-react`, `https://esm.sh/@itwin/itwinui-react@3`)
+  .replaceAll(`@itwin/itwinui-icons-react`, `https://esm.sh/@itwin/itwinui-icons-react@3`)
```

Comment:
`itwinui-icons-react`'s latest version is 2.8.0

```suggestion
  .replaceAll(`@itwin/itwinui-icons-react`, `https://esm.sh/@itwin/itwinui-icons-react@2`)
```

~But even with this, the demos that import from `https://esm.sh/@itwin/itwinui-icons-react@2` show a blank screen. E.g. `Tree`, `Tile`, `TransferList`, etc.. So I think this need to be investigated further.~ (This was an unrelated issue on my end.)